src/tests/routes/controllers/tag.test.js

```diff
@@ -0,0 +1,70 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+
+import app from '../../../server';
+import { generateToken } from '../../../utils/helpers.utils';
+import {
+  createTag,
+  updateTag,
+  removeTag,
+  viewTag,
+} from '../../../routes/controllers/tag.controller';
+
+// configure chai to use expect
+chai.use(chaiHttp);
+const { expect } = chai;
+
+const userToken = generateToken({
+  id: '5a6fab9c-5849-4be5-973c-5a371165cd57',
+  email: 'ecomje@gmail.com',
+});
+
+const articleId = '141f4f05-7d81-4593-ab54-e256c1006210';
+const generatedTags = 'faker.random.words(5).ddd'.split('.');
```

Comment:
Could you check out this line again? It seems you were calling a faker library but you made it a string.

Reply:
thanks, I will fix that.